src/main/java/com/example/demo/Order_item/Order_item.java

```diff
@@ -0,0 +1,47 @@
+package com.example.demo.Order_item;
```

Comment:
??? duplicate

Reply:
Remove this class

Reply:
Next time before you send me a MR check what you have here please

Reply:
done